langchain4j-azure-open-ai/src/main/java/dev/langchain4j/model/azure/AzureOpenAiEmbeddingModel.java

```diff
@@ -157,10 +157,10 @@ private Response<List<Embedding>> embedTexts(List<String> texts) {
         );
     }
 
-    private static Embedding from(List<Double> vector) {
+    private static Embedding from(List<Float> vector) {
```

Comment:
There is `Embedding.from(List<Float>)` which can be used here instead of manually iterating

Reply:
I'm using:
```
Embedding.from(vector.stream().map(Double::floatValue).collect(toList()));
```
Is it what was on your mind?

Reply:
I mean you do not need to convert `List<Float> vector` into `float[] langChainVector`, you can just do:
```
    private static Embedding from(List<Float> vector) {
        return Embedding.from(vector);
    }
```
Now there is no need in from() method, you can just do
`Embedding embedding = Embedding.from(embeddingItem.getEmbedding());`
in `embedTexts()`

Reply:
But you have literally changed `List<Double> vector` into `List<Float> vector` in this PR, I guess because `EmbeddingItem.getEmbedding()` now returns floats instead of doubles

Reply:
Sorry I was in the wrong branch!!! Let me test again